get-mods.sh

```diff
@@ -2,7 +2,21 @@
 # Icinga Web 2 Docker image | (c) 2020 Icinga GmbH | GPLv2+
 set -exo pipefail
 
-BRANCH="$1"
+usage() {
+	cat >&2 <<EOF
+Usage: get-mods.sh <release|snapshot>
+
+  release:  download the latest release version for all external modules
+  snapshot: download a snapshot/development version for all external modules
```

Comment:
"for"? I'd say "of".

Reply:
"for" as in for loop. But yes, "of" is probably better in natural language here.